run.go

```diff
@@ -238,6 +239,10 @@ var runCommand = &cli.Command{
 			Name:  "shm-size",
 			Usage: "Size of /dev/shm",
 		},
+		&cli.StringFlag{
```

Comment:
Perhaps StringSliceFlag to support multiple files

Reply:
Also, should be defined right after "env" flag

Reply:
> Perhaps StringSliceFlag to support multiple files

Agree with that